lib/yabeda/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Yabeda
-  VERSION = "0.1.3"
+  VERSION = "0.1.4"
```

Comment:
Let's not change versions in PR. I think it worth to change it to 0.2.0 maybe :smiley: , but I will do it when I will release new version to RubyGems

Reply:
Make sence :) I'll revert it

Reply:
Done